subsys/net/l2/openthread/Kconfig

```diff
@@ -70,6 +70,23 @@ config OPENTHREAD_SOURCES
 
 endchoice
 
+choice
+	prompt "OpenThread stack version"
+	help
+	  This option selects version of OpenThread stack.
+
+config OPENTHREAD_THREAD_VERSION_1_1
```

Comment:
ok for me, but please also consider extending existing options and choose the best one? i.e rename OPENTHREAD_SOURCES by adding version (it originally it even had version in its name but I was asked to remove it for now as there is only 1.1 now ;) ). Please keep in mind that after some merges, that I hope will happen today, in NCS new option appears named OPENTHREAD_LIBRARY_1_1 so using OPENTHREAD_SOURCES_1_1 OPENTHREAD_SOURCES_1_2 might be worth considering.